### PR TITLE
cachy-browser: 131.0.2-1

### PR DESCRIPTION
**Pull request by Stebalien**
- Also remove inetutils from makedepends per https://gitlab.archlinux.org/archlinux/packaging/packages/firefox/-/commit/1060ad6cb929f602230199d8029e243e098ce6e3.
- **WIP**: Update settings https://github.com/CachyOS/CachyOS-Browser-Common/pull/10.

### CODE REVIEW

File: cachy-browser/PKGBUILD [hunk 1]
```diff
@@ -91,18 +90,18 @@ install=cachy-browser.install
...
 
-_settings_commit=72b43a93b822ad06554d8dbe793b31fffef5aa74
-_common_commit=5dd46aafa8ad8b4bddc86a4febdab901e447c9a2
+_settings_commit=541156d4b673e18a565a76904ad1f3b726d1ccee
+_common_commit=7794576484c02a24075abe0a3a69bb443719b7ac
 source=(https://archive.mozilla.org/pub/firefox/releases/$pkgver/source/firefox-$pkgver.source.tar.xz{,.asc}
         $pkgname.desktop
         "git+https://github.com/cachyos/cachyos-browser-settings.git#commit=${_settings_commit}"
-        "git+https://github.com/cachyos/cachyos-browser-common.git#commit=${_common_commit}"
+        "git+https://github.com/Stebalien/cachyos-browser-common.git#commit=${_common_commit}"
```
[the remark this answers is not in the document]
Comment (Stebalien) [reply cachy-browser/PKGBUILD:98]: > WIP: Update settings https://github.com/CachyOS/CachyOS-Browser-Common/pull/10.

(that's why it's in draft)

[the remark this answers is not in the document]
Comment (1Naim) [reply cachy-browser/PKGBUILD:98]: Eh my bad, I didn't see it was in draft. Carry on 😃

[the remark this answers is not in the document]
Comment (Stebalien) [reply cachy-browser/PKGBUILD:98]: Np, someone trying to replace a dependency with a fork is sketch-af.

**Comment by ptr1337** [comment]
https://github.com/CachyOS/CachyOS-PKGBUILDS/commit/6a1c5950ca2a34b6c05c49d30ec30d80351f0891

Merged here :)

**Pull request closed without merging**